packages/mdc-line-ripple/README.md

```diff
@@ -8,34 +8,43 @@ path: /catalog/input-controls/line-ripple/
 
 # Line Ripple
 
-The line ripple is used to highlight user-specified text above it. When a line ripple is active, the line’s color and thickness vary.
+The line ripple is used to highlight user-specified text above it. When a line ripple is active, the line’s color and thickness changes.
```

Comment:
This was never changed to "user-specified input" from the earlier comment thread